example/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
-add_executable(inexor-vulkan-renderer-example main.cpp)
+add_executable(inexor-vulkan-renderer-example
+main.cpp
+# Windows resource file to set application icon and metadata
+${CMAKE_BINARY_DIR}/include/inexor/vulkan-renderer/resource.hpp
```

Comment:
I think this does not have to be explicit listed.

Reply:
I removed this file entirely.